plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveGlueCatalogConnectorSmokeTest.java

```diff
@@ -0,0 +1,240 @@
+/*
```

Comment:
Please refer to #14998. I don't think we want to add a new smoke test in this PR.

Reply:
fixed, thanks for help :)